bundle-size/api.js

```diff
@@ -110,15 +111,19 @@ function erroredCheckOutput(partialBaseSha) {
 function extraBundleSizesSummary(
   headSha,
   baseSha,
+  mergeSha,
   bundleSizeDeltasRequireApproval,
   bundleSizeDeltasAutoApproved,
   missingBundleSizes
 ) {
+  const compareUrl =
+    mergeSha ||
+    `https://github.com/ampproject/amphtml/compare/${baseSha}..${headSha}`;
   let output =
     '## Commit details\n' +
     `**Head commit:** ${headSha}\n` +
     `**Base commit:** ${baseSha}\n` +
-    `**Code changes:** https://github.com/ampproject/amphtml/compare/${baseSha}..${headSha}\n`;
+    `**Code changes:** ${compareUrl}\n`;
```

Comment:
Is there a way to explicitly make this a URL with `mergeSha`? (Or does GH linkify it under all circumstances?)

Reply:
Github links it just like it does for `headSha` and `baseSha`.